package.json

```diff
@@ -55,6 +55,7 @@
     "@cospired/i18n-iso-languages": "2.2.0",
     "axios": "0.26.1",
     "axios-cache-adapter": "2.7.3",
+    "axios-case-converter": "0.9.0",
```

Comment:
[curious] Should we be concerned about installing `axios-case-converter` if the consumer doesn't intend to use it? Maybe we should treat it more like an [optional dependency](https://betterprogramming.pub/what-are-npms-optional-dependencies-and-when-should-we-use-them-796a6a964e73)?

Reply:
[Chatted offline], we agreed that `axios-case-converter` should be a dependency since the intent is for consumers (mostly Open edX MFEs) to use it.